### PR TITLE
Add timer cycle with cron expressions

### LINKED ISSUE #1117: I can read about cycle timer definitions with cron expressions
With Zeebe 8.1, we added a new feature to specify cron expressions for a cycle timer definition of a timer start event (issue: https://github.com/camunda/zeebe/issues/https://github.com/camunda/zeebe/issues/9673, pr: https://github.com/camunda/zeebe/pull/9674).

We need to add documentation for it. This included:

 * [ ]  update BPMN docs: https://docs.camunda.io/docs/components/modeler/bpmn/timer-events/#time-cycle

### PR DESCRIPTION
**Pull request by lzgabel**
Adds documentation about the timer cycle with cron expressions.  E.g. `0 0 9-17 * * MON-FRI`

closes #1117

### CODE REVIEW

**Comment by lzgabel** [comment]
Hi @korthout, @christinaausley. Please take a moment to check this out. 🙇

**Comment by lzgabel** [comment]
It looks like something went wrong with our `linkcheck` script. 🤔 

image

**Comment by korthout** [comment]
Thanks @lzgabel I'll have a look at it tomorrow 🙇

File: docs/components/modeler/bpmn/timer-events/timer-events.md [hunk 1]
```diff
@@ -79,11 +79,13 @@ If the duration is zero or negative, the timer fires immediately.
 
 A cycle defined as ISO 8601 repeating intervals format; it contains the duration and the number of repetitions. If the repetitions are not defined, the timer repeats infinitely until it is canceled.
 It's possible to define a start time. By doing this, the timer triggers for the first time on the given start time. Afterwards, it will follow the interval as usual.
+Additionally, you can specify a time cycle using cron expressions. Please see the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions.
 
 - `R5/PT10S`: Every 10 seconds, up to five times
 - `R/P1D`: Every day, infinitely
 - `R/2022-01-01T10:00:00+02:00[Europe/Berlin]/P1D`: Every day infinitely, starting from January 1, 2022 at 10 a.m. UTC plus 2 hours
 - `R3/2022-04-27T17:20:00Z/P1D`: Every day up to three times, starting from April 27, 2022 at 5:20 p.m. UTC
+- `0 0 9-17 * * MON-FRI`: On the hour nine-to-five weekdays
```
Comment (christinaausley) [inline docs/components/modeler/bpmn/timer-events/timer-events.md:88]: I would rephrase this to clarify. "Every hour from 9–5 p.m. (enter timezone) Monday–Friday."

File: docs/components/modeler/bpmn/timer-events/timer-events.md [hunk 2]
```diff
@@ -79,11 +79,13 @@ If the duration is zero or negative, the timer fires immediately.
 
 A cycle defined as ISO 8601 repeating intervals format; it contains the duration and the number of repetitions. If the repetitions are not defined, the timer repeats infinitely until it is canceled.
 It's possible to define a start time. By doing this, the timer triggers for the first time on the given start time. Afterwards, it will follow the interval as usual.
+Additionally, you can specify a time cycle using cron expressions. Please see the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions.
```
Comment (christinaausley) [inline docs/components/modeler/bpmn/timer-events/timer-events.md:82]: You can remove "please" in our docs for conciseness. Simply "See the [CronExpression Tutorial](https://spring.io/blog/2020/11/10/new-in-spring-5-3-improved-cron-expressions) for additional information about using cron expressions." will do.

**Review by christinaausley — CHANGES_REQUESTED** [review]
See two comments for clarification of timezone and removing a few words for conciseness.

**Comment by akeller** [comment]
@lzgabel if you merge in `main` to your branch, it will likely resolve the build errors in the CI.

**Comment by lzgabel** [comment]
> @lzgabel if you merge in `main` to your branch, it will likely resolve the build errors in the CI.

Hi @akeller. Thanks for your reply. 💯 

---
Hi @korthout. Currently, my branch is checked out from `release_8_1`, do we need to merge the `main` branch to resolve the build errors in the CI?  🤔

**Comment by korthout** [comment]
@lzgabel Please rebase your changes on top of `main`. No need to target the `release_8_1` branch anymore. 🎉

**Comment by korthout** [comment]
I'll be merging that branch soon anyways 👍

File: docs/components/modeler/bpmn/timer-events/timer-events.md (hunk 1 above)
Comment (korthout) [reply docs/components/modeler/bpmn/timer-events/timer-events.md:88]: I agree, but would keep the part about `on the hour`.

Something like: `Every hour on the hour from 9-5 p.m. UTC Monday-Friday`

**Comment by lzgabel** [comment]
Hi @korthout, @christinaausley. I've fixed all review comments, please check this out. 🙇

**Review by korthout — APPROVED** [review]
Thanks @lzgabel LGTM 👍

**Comment by dlavrenuek** [comment]
谢谢 @lzgabel ! 👍

**Comment by lzgabel** [comment]
Hi @dlavrenuek. Thank you very much for your reply ❤️  (this is the first time I have received a reply in Chinese).

I almost thought that my Translate plugin was upgraded to the artificial intelligence version, it was able to automatically translate into Chinese. What's more funny is that I checked it several times. 🤣

**Comment by dlavrenuek** [comment]
Hi @lzgabel, I did not want to confuse you but this is indeed very amusing 😆 . To be honest I don't think that an artificial intelligence would be a match for my (bad) Chinese skills 👀 .

我真的喜欢你做的工作 。这个功能很有意思！
